ngraph/src/ngraph/coordinate_transform.cpp

```diff
@@ -29,6 +29,57 @@
 
 using namespace ngraph;
 
+namespace
+{
+    Strides default_strides(size_t n_axes) { return Strides(n_axes, 1); }
+    CoordinateDiff default_padding(size_t n_axes) { return CoordinateDiff(n_axes, 0); }
+    AxisVector default_axis_order(size_t n_axes)
+    {
+        AxisVector result(n_axes);
+        size_t n = 0;
+        std::generate(result.begin(), result.end(), [&n]() -> size_t { return n++; });
```

Comment:
May be simpler (faster?) to use `std::iota` here.

Reply:
I kept original code unchanged. Ok, I'll fix code according your comment.